components/card.css

```diff
@@ -4,6 +4,15 @@
     padding: 1rem;
     border-radius: 5px;
     position: relative;
+    border: 1px solid #e1e0e0;
```

Comment:
I personally like to keep the border color also in a variable, so in future, if the border color needs to be changed, developers have to modify only one variable.

Reply:
Thanks for the feedback Ruchi. I'll surely look into this.